server/filters/formatPrisonMovementEventFilter.ts

```diff
@@ -0,0 +1,23 @@
+import type { TimelineEvent } from 'viewModels'
+
+export default function formatPrisonMovementEventFilter(event: TimelineEvent): string {
```

Comment:
This is similar to the existing `formatTimelineEvent` filter, except that the formatted value needs to contain the prison name(s).